cmd/state/state.go

```diff
@@ -177,5 +177,8 @@ func getFlags(defaultFlags GlobalFlags, env map[string]string) GlobalFlags {
 	if _, ok := env["NO_COLOR"]; ok {
 		result.NoColor = true
 	}
+	if _, ok := env["K6_PROFILING_ENABLED"]; ok {
+		result.ProfilingEnabled = true
+	}
```

Comment:
Can you please add a test for this in the configuraiton consolidation tests as was previously requested in the issue.

Reply:
maybe I don't understand something, in my mind getConfigConsolidationTestCases uses if option has 2 ways for setting, isn't it?

Reply:
Yes and thsi can aso be set by cli flag

Reply:
I have seen a code - I have to check `--profiling-enabled` and `K6_PROFILING_ENABLED` together, rigth?